src/dashboard/Data/Browser/Browser.react.js

```diff
@@ -472,10 +585,71 @@ export default class Browser extends DashboardView {
       this.state.showRemoveColumnDialog ||
       this.state.showDropClassDialog ||
       this.state.showExportDialog ||
-      this.state.rowsToDelete
+      this.state.rowsToDelete ||
+      this.state.showAttachRowsDialog
     );
   }
 
+  showAttachRowsDialog() {
+    this.setState({
+      showAttachRowsDialog: true,
+      rowsToAttach: this.state.rowsToAttach || new List(),
+      attachRowsErrors: [],
+    });
+  }
+
+  cancelAttachRows() {
+    this.setState({
+      showAttachRowsDialog: false,
+    });
+  }
+
+  confirmAttachRows(objectIdsString) {
+    const relation = this.state.relation;
+    const query = new Parse.Query(relation.targetClassName);
+    const parent = relation.parent;
+    this.state.attachRowsErrors = [];
```

Comment:
right so when you want to change the value of any state `setState` should be used. If you're trying to clear the errors we should use `setState` rather than directly modifying it.